cfg/17.06/definitions.yaml

```diff
@@ -226,13 +236,14 @@ groups:
   - id: 2.1
     description: "Ensure network traffic is restricted between containers on the
     default bridge (Scored)"
+    failed_description: "Docker network com.docker.network.bridge.enable_icc is set to true"
     audit: "docker network ls --quiet | xargs docker network inspect --format '{{ .Name }}: {{ .Options }}'"
     tests:
       test_items:
-      - flag: "com.docker.network.bridge.enable_icc:false"
+      - flag: "com.docker.network.bridge.enable_icc"
```

Comment:
you'll see my comment in bench-common about this, but I'd love to see unit tests to validate this kind of change